tests/mozdef_util/test_elasticsearch_client.py

```diff
@@ -120,6 +120,27 @@ def test_search_no_results(self):
         assert results['hits'] == []
 
 
+class TestCloseIndex(ElasticsearchClientTest):
+
+    def test_index_close(self):
+        if pytest.config.option.delete_indexes:
+            self.es_client.create_index('test_index')
+        time.sleep(1)
+        indices = self.es_client.index_close('test_index')
+        assert indices is True
+
+
+class TestOpenIndex(ElasticsearchClientTest):
+
+    def test_index_open(self):
```

Comment:
We can improve these tests by closing an index, opening the index, and verifying that we can write an event.

Reply:
My previous comment stands, we should never write an event to any index that is not the current "events" tagged index. No events tagged index should ever be closed. This is not a use case we would ever pursue.

Reply:
Agreed!

Reply:
Done!